samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetectionML.Model/StopSignDetectionML.Model.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.3.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="MLModel.zip">
```

Comment:
I think this should be removed if we use the download mechanism.

Reply:
Fixed with https://github.com/dotnet/machinelearning-samples/pull/882/commits/d559ca315873ee6fad927f449f0a690e3d562f8b